ci/requirements-3.5_DOC.pip

```diff
@@ -0,0 +1 @@
+cssdecl
```

Comment:
add to ci/requirements-3.5.pip & requirements-2.7.pip as well (what you added here barely tests anything and only on windows).